Makefile

```diff
@@ -4,8 +4,8 @@ BABEL := babel --retain-lines
 RSYNC := rsync --archive
 VENV := .venv
 .DEFAULT_GOAL := help
-# Sets $(PAGESHOT_BACKEND) to http://localhost:10080 only if it isn't set
-PAGESHOT_BACKEND ?= http://localhost:10080
+# Sets $(SCREENSHOTS_BACKEND) to http://localhost:10080 only if it isn't set
+SCREENSHOTS_BACKEND ?= http://localhost:10080
 
```

Comment:
Also need to update the `zip` task ('page_shot', not 'pageshot'), currently fails with 

```sh
mv: rename web-ext-artifacts/page_shot*.zip to build/pageshot.zip: No such file or directory